### PR TITLE
feat: init project FE 

### PR DESCRIPTION
**Pull request by damminhtien**
+ Init base project
image

+ Simple register page
image

+ Simple login page
image

## Summary
+ Init base project with Typescript and Tailwind

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,54 +0,0 @@
-# Description
```
Comment (coleslaw) [inline README.md]: why delete?

**Review by coleslaw — CHANGES_REQUESTED** [review]
accidentally deleted code. Please revert them. 

Can you add a screenshot of the homepage to PR description?

**Comment by damminhtien** [comment]
@coleslaw It was a mistake of my teammate. He pushed code to my branch. I reverted to the commit of mine - the initial commit codebase of FE. 
We will push the homepage commit after this one is merged.

**Review by coleslaw — APPROVED** [review]
nice job @damminhtien

**Pull request merged**